lib/drivers/docs/tandem.md

```diff
@@ -33,7 +33,10 @@
     - `[x]` basal rate interval with a start time, duration "guessed" from settings, rate delivered, and an annotation re: the "guessed" duration OR
     - `[ ]` basal rate interval with a start time and rate, no (= zero) duration
 
-Device-specific? (Add any device-specific notes/additions here.)
+##### Device-specific? (Add any device-specific notes/additions here.)
+
+- we cannot add the name of the basal schedule on each scheduled basal rate interval at the moment, as that would require a full settings history
+- we cannot represent suppressed scheduled basals during suspended basals at the moment, as we require a full settings history to determine them for each segment of the basal schedule that the suspended basal intersects
 
```

Comment:
Maybe add a note explaining that Tandem doesn't _have_ manual temp basals, and that's why they aren't implemented?